clickhouse_search/models.py

```diff
@@ -724,7 +813,14 @@ def _save_table(
 
     class Meta:
         abstract = True
-
+        engine = CollapsingMergeTree(
+            'sign',
+            order_by=('project_guid', 'family_guid', 'key'),
+            partition_by='project_guid',
+            deduplicate_merge_projection_mode='rebuild',
+            index_granularity=8192,
+        )
+        projection = Projection('xpos_projection', order_by='xpos')
```

Comment:
Do we want an xpos projection for mito? I'm not sure it will really improve performance all that much given how mall the table is and how close all the xpos values are